es8/2017-03/mar-21.md

```diff
@@ -0,0 +1,1457 @@
+
+WH: Those are not the only choices. We could say "come back when you have improved it".
+
+ZB: Sure, but let's define the API before we move or should we say "this does not belong in ECMASCript" (do we move this forward or do we not)
+
+DH you will usually know if people think it shouldn't be in ECMASCript. That will be clear.
+
+YK: I think in general, that is a necessary, but not sufficient thing for stage 2. The norms of the committee is that the API is reasonably done as well as semantics.
+
+AWB: I was going to comment earlier. When the consensus is not to advance, it does not mean that it might pass in the future (without changes), It just means that people aren't ready to move along as it currently is. It might simply be a matter of coming back with more complete information.
+
+ZB: We are reviewing the proposal to move RelativeTimeFormat to stage 2.
+
+AWB: The relationship between this and the UnitFormat--I'd like to see more exploration of that. Are we going to do things with time types and duration types, and what might the impact of that be? These questions make me wonder whether this is ready for stage 2.
+
+YK: I think it's not ready for stage 2.
+
+BT: I don't think there's any rush for this for Stage 2 either. I'd like these to go into these as a bundle for 2018 for 402, but we have plenty of time for that.
+
+ZB: can I get clear direction form the committee whether we want to BLOCK the two formatters on duration types, or whether they can proceed without that.
+
+AWB: We're not hearing people say it's a blocking issue, more concern that it's not adequately explored or explained yet.
+
+ZB: What would be the recommendation, should I try to present a refined proposal separately or keep them as a bundle?
+
+WH: Whichever you do, some people will want the other, so do what makes sense.
+
+YK: Work together with the Duration proposal.
+
+CP: Let's keep the proposals separate; it's easy for me to integrate them separately.
+
+ZB: So ListFormat goes to Stage 2 and the others stay at Stage 1
+
+ZB: The last proposal I have is a proposal that I'm seeking stage 1 for. I presented it at the last meeting. It's a proposal for simplifying the way of formatting date and times using two formats `dateStyle` and `timeStyle` instead of doing them separately, we would actually like to use one of the four lengths of style `"short", "medium", "long" and "full". I think it's a pretty good representation. It makes it easier to use DateTimeFormat, but it also allows one more thing. It is language independent. For example, the "short date" specifiers are relative to the current locale. By using dateStyle or timeStyle the user hands of control of the formatting to the language. We expect this will be the most common way people use DateTimeFormat.
```

Comment:
Is it(\`) unnecessary?

![tc39-notes 2017-04-02 21-51-30](https://cloud.githubusercontent.com/assets/19714/24587288/9956ac60-17ee-11e7-8fe8-11d92d0c096f.png)